fe/lib/components/Questionnaire/QuestionnaireBuilder/QuestionnaireBuilder.tsx

```diff
@@ -65,16 +73,18 @@ export const QuestionnaireBuilder = ({
           </Column>
         </Columns>
 
-        <Card>
-          <Stack space="large">
-            <Heading level="3">GraphQL Output</Heading>
+        {showGraphqlOutput !== false && (
```

Comment:
```suggestion
        {showGraphqlOutput && (
```
Will checking truthy-ness work here?

Reply:
It's optional so that will make it default to `false` and change current behaviour.

Another option is having a `hideGraphqlOutput` property that works like a normal boolean but negative prop names are also annoying 🤔

Reply:
Ah I see - the property is optional; when the property exists, the value defaults to true